### ISSUE TITLE
Asus FX503V touchpad not detected

### ISSUE THREAD

**Issue by denijane** [gentoo/eudev#156]
Hi, I'm new here, so please apologise me if I'm posting in the wrong place, but I saw similar issue posted here. 
I bought a new laptop and the touchpad is not working at all. 
I'm with Sabayon Linux 4.9.0 (so Gentoo based), I tried also the next kernel but still no touchpad. 
In order to get my nvidia working and also the battery and temprature sensors, I have the option "acpi_osi! acipi_osi='Windows 2009' in my GRUB.  
But I have absolutely no touchpad - according to my system it doesn't exist. 
According to xinput:
 >sabayon # xinput
⎡ Virtual core pointer                          id=2    [master pointer  (3)]
⎜   ↳ Virtual core XTEST pointer                id=4    [slave  pointer  (2)]
⎜   ↳ ITE Tech. Inc. ITE Device(8910)           id=13   [slave  pointer  (2)]
⎜   ↳ Genius Optical Mouse                      id=11   [slave  pointer  (2)]
⎣ Virtual core keyboard                         id=3    [master keyboard (2)]
    ↳ Virtual core XTEST keyboard               id=5    [slave  keyboard (3)]
    ↳ Power Button                              id=6    [slave  keyboard (3)]
    ↳ Video Bus                                 id=7    [slave  keyboard (3)]
    ↳ Video Bus                                 id=8    [slave  keyboard (3)]
    ↳ Power Button                              id=9    [slave  keyboard (3)]
    ↳ Sleep Button                              id=10   [slave  keyboard (3)]
    ↳ USB2.0 HD UVC WebCam                      id=12   [slave  keyboard (3)]
    ↳ Asus WMI hotkeys                          id=14   [slave  keyboard (3)]
    ↳ AT Translated Set 2 keyboard              id=15   [slave  keyboard (3)]
    ↳ ITE Tech. Inc. ITE Device(8910)           id=16   [slave  keyboard (3)]

After googling it, it seems my touchpad is detected as :  ITE Tech. Inc. ITE Device(8910):

>sabayoni # cat /proc/bus/input/devices
I: Bus=0003 Vendor=0b05 Product=1869 Version=0110
N: Name="ITE Tech. Inc. ITE Device(8910)"
P: Phys=usb-0000:00:14.0-8/input0
S: Sysfs=/devices/pci0000:00/0000:00:14.0/usb1/1-8/1-8:1.0/0003:0B05:1869.0002/input/input9
U: Uniq=
H: Handlers=sysrq kbd event8 leds 
B: PROP=0
B: EV=12001f
B: KEY=3007f 1 ffffffffffffffff ffffffffffffffff ffffffffffffffff ffffffffffffffff 130c130b17c007 ffbf7bfad941dfff febeffdfffefffff fffffffffffffffe
B: REL=40
B: ABS=ffffff0100000000
B: MSC=10
B: LED=1f

So it's not recognised as a touchpad at all. I spent a lot of time googling the problem and I didn't find a solution, so it seems to me maybe the driver should be updated? I have no idea what the touchpad manifacturer is though, becausee I don't have windows installed to check. But according to [this](https://www.queryoverflow.gdn/query/asus-fx503vd-elan1200-touchpad-not-working-smoothly-3_981183.html), it is ELAN 1200.

**Comment by blueness** [comment]
Sabayon Linux 4.9.0 uses systemd, not eudev.  This is not a problem we can fix.

**Comment by denijane** [comment]
Ok, could you tell me then where to post the bug? Becuase it's clearly a problem if the touchpad is not detected at all.

**Comment by blueness** [comment]
its a sabayon issue.  start with their forums.

**Comment by denijane** [comment]
I did that already, but they couldn't help me. And besides, I don't think it's their issue, since I tested about 5 distros (Fedora, Xubunto, Manjaro, Mint, GhostBSD and Scientific Linux) and not even one managed to detect my touchpad. So obviously, it's a problem of the drivers and/or kernel, which do not support this touchpad. Also Sabayon is Gentoo based, so Gentoo users will also benefit from solving the issue.
